backend/src/main/java/com/benine/backend/http/FocussingHandler.java

```diff
@@ -45,15 +45,18 @@ public void handle(HttpExchange exchange) throws IOException {
     FocussingCamera focusCam = (FocussingCamera) cam;
     String autoOn = parsedURI.getValue("autoFocusOn");
     String setPos = parsedURI.getValue("position");
+    String speed = parsedURI.getValue("speed");
     try {
       if (autoOn != null) {
         boolean autoOnBool = Boolean.parseBoolean(autoOn);
         focusCam.setAutoFocusOn(autoOnBool);
       }
       if (setPos != null) {
         focusCam.setFocusPos(Integer.parseInt(setPos));
+      } if(speed != null) {
+        focusCam.moveFocus((int)(50 + (4.45*Integer.parseInt(speed))));
```

Comment:
Yes, I think it would be better to do it client side, because otherwise it could get confusing in the future when you change one thing and it's maped a few times to another domain.